test/acceptance/integration/tools/general_utils.py

```diff
@@ -291,7 +295,19 @@ def start_cb(parms):
     set_ssh_config(localhost)
     if world.cb_pid != '':
         stop_cb()
-    getattr(api, runner)('{bin_path} {parms} >> /tmp/cb_acceptance_test.log'.format(bin_path=config['bin_path'], parms=parms))
+    if config['log_path'] == '':
+        log_path = '/tmp/acceptance'
+    else:
+        log_path = config['log_path']
+    getattr(api, runner)('mkdir -p {log_path}'.format(log_path=log_path))
+    if config['pid_file'] == '':
+        pid_path = '/tmp/acceptance/contextBroker.pid'
+    else:
+        pid_path = config['pid_file']
+    getattr(api, runner)('mkdir -p {pid_dir}'.format(pid_dir=pid_path[:pid_path.rfind('/')]))
+    command = '{bin_path} {parms} -logDir {log_path} -pidpath {pid_path} >> /tmp/cb_acceptance_test.log'.\
```

Comment:
Same problem here. An overall grep for "/tmp" in the code could help to locate all cases.